### ISSUE TITLE
BUG: mysql-formatter adds spaces when using backticks

### ISSUE THREAD

**Issue by ImagineItBetter** [beekeeper-studio/beekeeper-studio#814]
**Describe the bug**
In a query window inside Beekeeper Studio, connected to a mysql database, if I select "Format Query," whitespace is being added to backticks surrounding words. 

**To Reproduce**
1. Connect to a mysql database inside Beekeeper
2. Create a new query, using backticks, ex: ``SELECT * FROM `table` WHERE `id` = 1``
3. Right-click inside the window and choose "Format Query"
4. Formatting the query adds spaces between text and backticks: ``SELECT * FROM ` table ` WHERE ` id ` = 1``

**Expected behavior**
Formatting a MySQL query should not add spaces before and after backticks surrounding words.

**Version Information (please complete the following information):**
 - OS: Windows 10, Version 10.0.19042, Build 19042
 - App Version: 2.1.4

**Additional context**
Beekeeper Studio uses mysql-formatter to format queries. The default language that mysql-formatter uses to format is 'sql' if the language is not specified. This seems to be how Beekeeper is using it in `beekeeper-studio/apps/studio/src/components/TabQueryEditor.vue`. 

Mysql-formatter has a demo page that this can also be replicated on: https://zeroturnaround.github.io/sql-formatter/

Go to the page, add backticks to the query on the left, and change the format to see the different options at a glance. The default option on the page, SQL, adds backticks. Switching to MySQL (and some others) does not add the whitespace.

This issue on mysql-formatter explains the bug in mysql-formatter seemingly causing the problem: https://github.com/zeroturnaround/sql-formatter/issues/139

**Suggestions / Brainstorm**
According to the issue raised on mysql-formatter, the issue seems to be remedied if you roll back to v3.1.0. This might be the quickest, if not the best way to fix it, depending on what that rollback breaks.

Another option might be changing the default used by Beekeeper when formatting to MySQL. This would at the very least solve my direct issue, but might break things for others.

The last option I considered was, Beekeeper could (if it isn't already) feed in the type of format the person is using as the format type. It doesn't seem to be doing this with mysql while I'm in a mysql database, but I'm very new to Beekeeper so I may be missing it.

**Pull Request**
I'd like to make an attempt at fixing this at some point if it is decided to be a bug. I'm not familiar with the project, but let me know when you're able if any of those fixes are the right path to continue down. If it is easy to toss out an idea of what should be done, even vaguely, I'd love to see if I could make it happen.

**Comment by WebMechanic** [comment]
I wanted to collect the various create statements into a .sql file for later use.

When I right-click a table and pick "SQL: create" from the popup menu this is what I got (excerpt): 😱
```sql
CREATE TABLE ` fubar ` (
  ` id ` int(10) unsigned NOT NULL AUTO_INCREMENT,
  ` title ` varchar(120) COLLATE utf8_swedish_ci NOT NULL,
  PRIMARY KEY (` id `),
  KEY ` IDX_TITLE ` (` title `)
)
```
should be
```sql
CREATE TABLE `fubar` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `title` varchar(120) COLLATE utf8_swedish_ci NOT NULL,
  PRIMARY KEY (`id`),
  KEY `IDX_TITLE` (`title`)
)
```

Whatever the technical background for this might be, @ImagineItBetter was so kind to elaborate about, this is a bummer and of no use.

Please fix 🙏🏻
Cheers

**Comment by rathboma** [comment]
Fixed! #816
